arcgis-ios-sdk-samples/Display information/Add graphics with symbols/GraphicsWithSymbolsViewController.swift

```diff
@@ -72,160 +72,160 @@ class GraphicsWithSymbolsViewController: UIViewController {
         let buoyGraphic4 = AGSGraphic(geometry: buoy4Loc, symbol: buoyMarker, attributes: nil)
         
         //add the graphics to the graphics overlay
-        graphicsOverlay.graphics.addObjectsFromArray([buoyGraphic1, buoyGraphic2, buoyGraphic3, buoyGraphic4])
+        graphicsOverlay.graphics.addObjects(from: [buoyGraphic1, buoyGraphic2, buoyGraphic3, buoyGraphic4])
     }
     
-    private func addText(graphicsOverlay:AGSGraphicsOverlay) {
+    fileprivate func addText(_ graphicsOverlay:AGSGraphicsOverlay) {
         
-        let wgs84 = AGSSpatialReference.WGS84()
+        let wgs84 = AGSSpatialReference.wgs84()
         let bassLocation = AGSPoint(x: -2.640631, y: 56.078083, spatialReference: wgs84)
         let craigleithLocation = AGSPoint(x: -2.720324, y: 56.073569, spatialReference: wgs84)
         
         //create text symbols
-        let bassRockSymbol = AGSTextSymbol(text: "Bass Rock", color: UIColor(red: 0, green: 0, blue: 230/255.0, alpha: 1), size: 10, horizontalAlignment: AGSHorizontalAlignment.Left, verticalAlignment: AGSVerticalAlignment.Bottom)
+        let bassRockSymbol = AGSTextSymbol(text: "Bass Rock", color: UIColor(red: 0, green: 0, blue: 230/255.0, alpha: 1), size: 10, horizontalAlignment: AGSHorizontalAlignment.left, verticalAlignment: AGSVerticalAlignment.bottom)
         
-        let craigleithSymbol = AGSTextSymbol(text: "Craigleith", color: UIColor(red: 0, green: 0, blue: 230/255.0, alpha: 1), size: 10, horizontalAlignment: AGSHorizontalAlignment.Right, verticalAlignment: AGSVerticalAlignment.Top)
+        let craigleithSymbol = AGSTextSymbol(text: "Craigleith", color: UIColor(red: 0, green: 0, blue: 230/255.0, alpha: 1), size: 10, horizontalAlignment: AGSHorizontalAlignment.right, verticalAlignment: AGSVerticalAlignment.top)
         
         //define a graphic from the geometry and symbol
         let bassRockGraphic = AGSGraphic(geometry: bassLocation, symbol: bassRockSymbol, attributes: nil)
         let  craigleithGraphic = AGSGraphic(geometry: craigleithLocation, symbol: craigleithSymbol, attributes: nil)
         
         //add the text to the graphics overlay
-        graphicsOverlay.graphics.addObject(bassRockGraphic)
-        graphicsOverlay.graphics.addObject(craigleithGraphic)
+        graphicsOverlay.graphics.add(bassRockGraphic)
+        graphicsOverlay.graphics.add(craigleithGraphic)
     }
     
-    private func addBoatTrip(graphicsOverlay:AGSGraphicsOverlay) {
+    fileprivate func addBoatTrip(_ graphicsOverlay:AGSGraphicsOverlay) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage